cmd/util/defaults.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2021 The cert-manager Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"time"
+)
+
+const (
+	DefaultLeaderElect                 = true
+	DefaultLeaderElectionNamespace     = "kube-system"
```

Comment:
We should talk about whether this is a good default (see https://github.com/jetstack/cert-manager/issues/4102) and also whether this could conceivably break things on upgrade....might it mean that a new and an old cainjector process is briefly reconciling because they each use a different leader-election lock resource....and does that matter?
Also, might there be people who are deliberately running multiple  `cainjector` processes in separate namespaces and relying on its current use of the "current namespace" as the default for leader-election.

Reply:
See my comment on [cmd/cainjector/app/start.go](https://github.com/jetstack/cert-manager/pull/4359/files#r688638398), this is already explicitly set as `kube-system` in the helm chart and manifests.

Reply:
I see. 

https://github.com/jetstack/cert-manager/blob/54c66769bc253f4f786fe92cded132dab9de65fb/deploy/charts/cert-manager/templates/cainjector-deployment.yaml#L62-L63

https://github.com/jetstack/cert-manager/blob/54c66769bc253f4f786fe92cded132dab9de65fb/deploy/charts/cert-manager/values.yaml#L23-L25

I hadn't realised that cainjector was also, already using kube-system, so your change should not make any difference and we can address the other issue of whether `kube-system` is the correct default in another PR. Fine.